infra/gravity/cluster_status.go

```diff
@@ -90,20 +83,23 @@ func (c *TestContext) Status(nodes []Gravity) ([]*GravityStatus, error) {
 		go func(n Gravity) {
 			status, err := n.Status(ctx)
 			errC <- err
-			valueC <- status
+			valueC <- *status
```

Comment:
This needs to be done conditionally.  If err, this will result in a nil pointer dereference.

Reply:
Fixed in d540cb5.